service/dap/server_test.go

```diff
@@ -4562,41 +4555,41 @@ func TestBadLaunchRequests(t *testing.T) {
 			"Failed to launch: The program attribute is missing in debug configuration.")
 
 		// Bad "args"
-		client.LaunchRequestWithArgs(map[string]interface{}{"mode": "exec", "program": fixture.Path, "args": nil})
+		client.LaunchRequestWithArgs(map[string]interface{}{"mode": "exec", "program": fixture.Path, "args": "foobar"})
```

Comment:
nil is a perfectly fine (zero) value for []string type so this will be accepted as a valid launch config.
Is there a reason that args: nil should be considered a bad value?